.storybook/preview.js

```diff
@@ -16,6 +16,10 @@ export const parameters = {
     disable: true,
     grid: { disable: true }
   },
+  controls: {
+    hideNoControlsWarning: true,
+    sort: 'alpha'
```

Comment:
Hmm this will change up the controls quite a bit.. and would favor alphabetical over a curated set (unless manually overridden per story). Is this to standardize controls?

Reply:
@Francois-Esquire I will remove this sorting option for now. With the forthcoming Storybook refactor, this will be re-added to standardize controls – all of which should be tied to props and consistently sorted similar to what we see on the website. Curated, custom controls will be gathered under specific sections. Stay tuned.